src/model.ts

```diff
@@ -54,6 +54,14 @@ export class Annotation {
   }
 }
 
+export class Comment {
+  readonly text: string
+
+  constructor(name: Name){
+    this.text = name
+  }
+}
+
```

Comment:
```suggestion
export class Comment {
  constructor(public readonly text: Name){}
}
```

usando el syntactic sugar de TS...

Reply:
Esto al final nunca lo use y me olvide de borrarlo 😅 Los comentarios quedaron como annotations